### ISSUE TITLE
SabanYelkeni: Behavior Strategy Implementation, 3 weeks

### ISSUE THREAD

**Issue by sabanyelkenci** [HackYourFutureBelgium/class-13-14#252]
> [module repository](https://github.com/HackYourFutureBelgium/behavior-strategy-implementation)

## Week 1

- [ ] [./array-methods](https://github.com/HackYourFutureBelgium/behavior-strategy-implementation/blob/master/array-methods)
- [ ] [./hoisting](https://github.com/HackYourFutureBelgium/behavior-strategy-implementation/blob/master/hoisting)
- [ ] [./about-testing](https://github.com/HackYourFutureBelgium/behavior-strategy-implementation/blob/master/about-testing)
- [ ] [ ./parsons-problems](https://github.com/HackYourFutureBelgium/behavior-strategy-implementation/blob/master/parsons-problems)

### Sunday Prep Work

- [ ] Practice Testing Functions from ./about-testing/exercises
- [ ] Read through the steps of Function Design and study the example:
          - [ ] ./function-design/README.md
          - [ ] ./function-design/examples/reverse-a-string

---- END CHECK-IN TEMPLATE ----

-->

**Comment by sabanyelkenci** [comment]
### Check-In W1

#### I Need Help With:

#### What went well? Edabit is really useful platform to understand the different solutiions and I like it 

#### What went less well? I have to spend much more time to exercises

#### Lessons Learned

**Comment by sabanyelkenci** [comment]
Check-In W2
I Need Help With: 
What went well? : I repeated the previous lessons and Javascript now makes more sense
What went less well?  Since my Dutch lessons progressed so intensely at these times, I could spend a little bit of time for programming
Lessons Learned: do homeworks on time

**Comment by sabanyelkenci** [comment]
Check-In W3
I Need Help With:
What went well?  I wrote the function and test cases and I run the test. All the tests are passed.  I like group working. So I am more motivated with group working. 
What went less well? 
Lessons Learned

**Comment by Lizhen2910** [comment]
@sabanyelkenci I totally agree, group work is much more motivating. Thanks for the checking-in!
